### PR TITLE
Trackstack case study

### PR DESCRIPTION
**Pull request by CezzaineZ**
## Summary
Copy doc: https://docs.google.com/document/d/1tQKqjm4Qc-G13u0GsVjfUM4a4NAYnltygFc6Xp4Iwzg/edit

### Blog Release Checklist

- \[ ] Release on: \<10/16/24>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: case-study-trackstack.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+published: true
+slug: case-study-trackstack
+ogImage: https://us-east-1.storage.xata.sh/uq2d57ceoqafb0197c976kvrugevhlf8
+---
+
+<ArticleImage
+  src="https://us-east-1.storage.xata.sh/uq2d57c1r5efeldl3rbc8l19901jii30"
+  width="800"
+  height="405"
+  alt="Savas labs hero image"
```
Comment (alexfrancoeur) [inline case-study-trackstack.mdx:19]:
```suggestion
  alt="Trackstack hero image"
```

File: case-study-trackstack.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+## Results and impact
+
+Since migrating to Xata, Trackstack has experienced significant improvements in both platform performance and operational efficiency:
+
+- **67% faster query speeds:** Trackstack reduced the average query time for their largest queries from 6 seconds to 2 seconds, greatly improving the user experience.
+- **32 hours saved per month:** Xata’s fully managed infrastructure freed up 8 hours of developer time each week, allowing the team to focus on building new features rather than maintaining the database.
+- **Simplified and predictable billing:** Xata’s cluster-based billing structure provided Trackstack with clear, predictable costs, making it easier to plan for growth without worrying about unpredictable charges for database metrics.
+
+> Xata not only improved our query performance by 67%, but also saved us significant developer time each week, allowing us to focus on feature development. The simplified billing structure has made it easier to manage costs as we grow.
+> ––_Cameron Jenkinson, Co-Founder at Trackstack_
```
Comment (alexfrancoeur) [inline case-study-trackstack.mdx:80]:
```suggestion
> Xata not only improved our query performance by 67%, but also saved us significant developer time each week, allowing us to focus on feature development. The simplified billing structure has made it easier to manage costs as we grow.\
> _Cameron Jenkinson, Co-Founder at Trackstack_
```

File: case-study-trackstack.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Trackstack’s expanding user base, which includes prominent figures like Jamie Jones and Bob Sinclair, underscores its rapid growth and adoption within the electronic music industry. As the platform continues to scale, it remains a key player in helping music professionals manage their careers while providing innovative tools that simplify and enhance music management and distribution.
+
+![Trackstack UI](/images/trackstack.png)
+
+## Challenges and the task ahead
+
+As Trackstack’s user base grew to over 60,000 music professionals, the team began facing significant challenges with their existing database infrastructure. PlanetScale, their initial choice for managing large data volumes, could no longer keep up with Trackstack's rapid growth. Queries on large datasets took up to 6 seconds to load, causing significant delays for users who needed fast access to music libraries. This affected the responsiveness of Trackstack’s platform, leading to frustrations among DJs, producers, and label managers who rely on swift workflows.
```
Comment (alexfrancoeur) [inline case-study-trackstack.mdx:40]:
```suggestion
As Trackstack’s user base grew to over 60,000 music professionals, the team began facing significant challenges with their existing database infrastructure. [PlanetScale](https://planetscale.com/), their initial choice for managing large data volumes, could no longer keep up with Trackstack's rapid growth. Queries on large datasets took up to 6 seconds to load, causing significant delays for users who needed fast access to music libraries. This affected the responsiveness of Trackstack’s platform, leading to frustrations among DJs, producers, and label managers who rely on swift workflows.
```

**Review by alexfrancoeur — APPROVED** [review]
Added a few minor tweaks, but LGTM! I'm not sure if it helps us if we link to PlanetScale or not. What do you think about highlighting the migration off of planetscale in the social a bit more? I feel like that's the main message we want people to takeaway from this blog

File: case-study-trackstack.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Xata’s support for PostgreSQL also made the transition smoother by allowing Trackstack to retain their existing data structures. Additionally, Xata's simplified billing model—based on database cluster size rather than metrics like rows read or storage volume—made it easier for Trackstack to predict and control costs as their platform grew.
+
+> After moving to Xata, our largest query times dropped from 6 seconds to 2 seconds, which makes a huge difference to our users. Plus, the predictable pricing model has given us clarity and control over our database costs as we scale.
+> ––_Cameron Jenkinson, Co-Founder at Trackstack_
```
Comment (CezzaineZ) [inline case-study-trackstack.mdx:55]:
```suggestion
> _Cameron Jenkinson, Co-Founder at Trackstack_
```

File: case-study-trackstack.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+Xata’s support for PostgreSQL also made the transition smoother by allowing Trackstack to retain their existing data structures. Additionally, Xata's simplified billing model—based on database cluster size rather than metrics like rows read or storage volume—made it easier for Trackstack to predict and control costs as their platform grew.
+
+> After moving to Xata, our largest query times dropped from 6 seconds to 2 seconds, which makes a huge difference to our users. Plus, the predictable pricing model has given us clarity and control over our database costs as we scale.
```
Comment (CezzaineZ) [inline case-study-trackstack.mdx:54]:
```suggestion
> After moving to Xata, our largest query times dropped from 6 seconds to 2 seconds, which makes a huge difference to our users. Plus, the predictable pricing model has given us clarity and control over our database costs as we scale./
```

**Review by CezzaineZ — COMMENTED** [review]
adjusting quote

File: case-study-trackstack.mdx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+Xata’s support for PostgreSQL also made the transition smoother by allowing Trackstack to retain their existing data structures. Additionally, Xata's simplified billing model—based on database cluster size rather than metrics like rows read or storage volume—made it easier for Trackstack to predict and control costs as their platform grew.
+
+> After moving to Xata, our largest query times dropped from 6 seconds to 2 seconds, which makes a huge difference to our users. Plus, the predictable pricing model has given us clarity and control over our database costs as we scale./
```
Comment (CezzaineZ) [inline case-study-trackstack.mdx:54]:
```suggestion
> After moving to Xata, our largest query times dropped from 6 seconds to 2 seconds, which makes a huge difference to our users. Plus, the predictable pricing model has given us clarity and control over our database costs as we scale.\
```

**Pull request merged**